sympy/geometry/polygon.py

```diff
@@ -402,6 +403,59 @@ def centroid(self):
             cy += v*(y1 + y2)
         return Point(simplify(A*cx), simplify(A*cy))
 
+    def second_moment_of_area(self, p=None):
+        """Second and product moment of area of polygon.
+
+        Parameters
+        ==========
+
+        p : Point, None
+            p is the point about which second moment of area is to be find. Default value is None
```

Comment:
"is to be found"

Reply:
"Default value is None" > "If None, the point will be taken as the centroid of the polygon."

Reply:
find > found

Reply:
Done